conf/app.routes

```diff
@@ -1,3 +1,3 @@
 # microservice specific routes
 
-GET        /hello-world             uk.gov.hmrc.transitmovementsconverter.controllers.MicroserviceHelloWorldController.hello
+POST       /convert/:messageType             uk.gov.hmrc.transitmovementsconverter.controllers.ConverterController.convert(messageType: MessageType[_])
```

Comment:
The validator has a different endpoint format /transit-movements-validator/messages/:messageType/validation - do we want to replicate this format for the converter?

Reply:
/messages/:messageType/convert/xmlToJson and /messages/:messageType/convert/jsonToXml? I'm not opposed to it. It would be clearer to understand what's happening by just looking at the endpoint..

Reply:
Well, if we're sticking to RESTful principles, it'd probably be something more like

`/messages/:messageType/convert/xml` for getting XML
`/messages/:messageType/convert/json` for getting Json

We specify what we're converting from in the content-type header. In theory, we could use the Accept header to specify what we want to convert to, rather than the URL.

I wonder what @crystalmark and @RobertLaverick think.